projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -45,6 +47,14 @@ const Index = () => {
 		recordBoostEvent( 'critical_css_link_clicked', {} );
 	};
 
+	const upgradeClickHandlerModal = useCallback( () => {
+		// record event for opening a modal
+		recordBoostPixelEvent( 'jetpack_boost_upgrade_button_clicked', {
+			placement: 'product-page',
+			context: 'jetpack-boost',
+		} );
+	}, [] );
```

Comment:
Just noting this in case it was missed.

There's already a tracking event for clicking on the CTA in the `UpgradeCTA` component:

https://github.com/Automattic/jetpack/blob/cea499ae5478d48ed5c80be65a3d61298efe2164/projects/plugins/boost/app/assets/src/js/features/upgrade-cta/upgrade-cta.tsx#L30

Reply:
Having a different `event` triggered on open makes sense to me, along with the event that is triggered on the CTA click. I'll update the event name.